src/test/rustdoc-ui/range-pattern.rs

```diff
@@ -0,0 +1,3 @@
+// check-pass
+
+fn func(0u8..=255: u8) {}
```

Comment:
Can you also add a test that this is displayed correctly in the generated docs? It would go in `src/test/rustdoc`, see `fn-pointer-arg-name.rs` for an example.

Reply:
Adding the test right now, sorry for the delay, took some holiday time off of productivity :P

Reply:
No problem at all, glad you enjoyed the holiday :)